src/drivers/intel/cavs/dmic.c

```diff
@@ -853,18 +863,17 @@ static int configure_registers(struct dai *dai, struct dmic_configuration *cfg,
 
 	/* pdata is set by dmic_probe(), error if it has not been set */
 	if (!pdata) {
-		trace_dmic_error("cfr");
+		trace_dmic_error("configure_registers() error: pdata not set");
 		return -EINVAL;
 	}
 
 	/* Sanity checks */
 	if (dmic->num_pdm_active > DMIC_HW_CONTROLLERS) {
-		trace_dmic_error("num");
+		trace_dmic_error("configure_registers() error: sanity error");
```

Comment:
Yes though more precisely, the requested PDM controllers count exceeds platform capability.

Reply:
Fixed